tools/packaging/kata-deploy/local-build/Makefile

```diff
@@ -29,6 +29,7 @@ all: serial-targets \
 	kernel-gpu \
 	kernel-gpu-snp-tarball \
 	kernel-gpu-tdx-experimental-tarball \
+	kernel-sev-tarball \
```

Comment:
We're trying to keep this alphabetically sorted, although the `gpu` targets are already not doing that. :-/
Zvonko will take care of sorting those out in his next iteration, but I'd like to ask you to have the kernel-sev-tarball built before the kernel-tdx-experimental-tarball.